crates/forge/tests/cli/test_cmd.rs

```diff
@@ -1919,3 +1919,99 @@ contract CounterRevertTest is DSTest {
 ...
 "#]]);
 });
+
+forgetest_init!(should_generate_junit_xml_report, |prj, cmd| {
+    prj.wipe_contracts();
+    prj.add_source(
+        "JunitReportTest.t.sol",
+        r#"
+import "forge-std/Test.sol";
+contract AJunitReportTest is Test {
+    function test_junit1() public {
```

Comment:
can we name these for what they're testing for, eg revert, skip ...

Reply:
yep, https://github.com/foundry-rs/foundry/pull/8852/commits/143d0686f8a2efe4288db2575858d0f66e8afe3b